.gitignore

```diff
@@ -34,4 +34,15 @@
 !/app/assets/builds/.keep
 
 /config/credentials/production.key
-/.env
\ No newline at end of file
+/.env
+
+# Ignore Bust A Gem VSCode extension ctags generation.
+/TAGS
+
+# Ignore vscode files.
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
```

Comment:
Other Rubyists I work with are using VSCode, so that's what I've ended up with as my IDE. I'd say it's common-enough these days to warrant these innocuous additions.

Reply:
I think these editor-specific ignore directives would be better in a developers [local/global gitignore file](https://git-scm.com/docs/gitignore) so that we don't have to worry about keeping them up to date as editors change.